news/_posts/core-weekly/2020-06-15-coreweekly-24-2020.md

```diff
@@ -0,0 +1,396 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week XXXX of 2020"
```

Comment:
New week number?

Reply:
```suggestion
title:  "PrestaShop Core Weekly - Week 24 of 2020"
```
thanks for the catch.
is it something we can automatize, since this is a known information with the --week option?

Reply:
Yes, but it's like using GitHub Desktop, too complicated for me :trollface: